contributing/development/core_and_modules/object_class.rst

```diff
@@ -71,13 +71,13 @@ Registering functions is one:
 
 .. code-block:: cpp
 
-    ClassDB::bind_method(D_METHOD("methodname", "arg1name", "arg2name"), &MyCustomMethod);
+    ClassDB::bind_method(D_METHOD("methodname", "arg1name", "arg2name", "arg3name"), &MyCustomType::method);
 
-Default values for arguments can be passed in reverse order:
+Default values for arguments can be passed:
```

Comment:
What "reverse order" tried to convey is that it applies to the X last arguments matching the X default values provided. One could expect otherwise that it would set default values for `arg1name` and `arg2name` below, and that `arg3name` would be `null`, or that it would fail compiling.

I think we might still want to try to make this clear here. But I'm not sure how to phrase it (reverse order was the wrong term for sure).

Reply:
Also ending the sentence like that is abrupt and doesn't work without "in reverse order" :)

We can adjust this sentence to say something like "Default values for arguments can also be passed as parameters at the end:". And after the sample we could add an explanation about the order more verbosely. Something like...

"Default values must be supplied in the same order as function's declared arguments are. You must skip all required arguments, and then provide default values for the optional ones."

Reply:
Updated, see what you think, the last part comparing to c++ is optional but thought it would be helpful